src/scripts/main.js

```diff
@@ -1,3 +1,430 @@
+        'Congrats! The office was successfully changed.',
+        'success');
+    };
+
+    cellOffice.onkeydown = (enter) => {
+      if (enter.key === 'Enter') {
+        cellOffice.onblur();
+      }
+    };
+  } else {
+    const cellInput = document.createElement('input');
+
+    cellInput.className = 'cell-input';
+    cellInput.innerHTML = td.innerHTML;
+    td.innerHTML = '';
+    td.append(cellInput);
+    cellInput.focus();
+
+    switch (td.cellIndex) {
+      case 0:
+      case 1:
+        cellInput.setAttribute('type', 'text');
+        break;
+
+      case 3:
+      case 4:
+        cellInput.setAttribute('type', 'number');
+        break;
+    }
+
+    cellInput.onblur = (e) => {
+      validate(td.cellIndex, cellInput);
+      editEnd(editingArea.elem, isCorrect);
+      changeIsCorrect();
+    };
+
+    cellInput.onkeydown = (enter) => {
+      if (enter.key === 'Enter') {
+        cellInput.onblur();
+      }
+    };
+  }
+};
+
+function validate(index, cell) {
+  switch (index) {
+    case 0:
+    case 1:
+      if (/\d/.test(cell.value)
+        || cell.value.trim().length < nameLength) {
+        isCorrect = false;
+
+        pushNotification(
+          580,
+          10,
+          'Enter a correct data!',
+          'This field must contain at least 4 letters and not include numbers.',
+          'error');
+      }
+
+      break;
+
+    case 3:
+      if (cell.value < 18
+        || cell.value > 90) {
+        if (cell.value === '') {
+          isCorrect = false;
+
+          return;
+        }
+
+        isCorrect = false;
+
+        pushNotification(
+          580,
+          10,
+          'Enter a correct age!',
+          'The age must be 18+ and less than 90.',
+          'error');
+      }
+
+      break;
+
+    case 4:
+      if (+cell.value <= 0) {
+        if (cell.value === '') {
+          isCorrect = false;
+
+          return;
+        }
+
+        isCorrect = false;
+
+        pushNotification(
+          580,
+          10,
+          'Enter a correct salary!',
+          'The salary must be more than 0.',
+          'error');
+      }
+
+      break;
+  }
+}
+
+function editEnd(td, isOk) {
+  if (isOk) {
+    switch (td.cellIndex) {
+      case 4:
+        td.innerHTML = `${(+td.value)}`;
+
+        pushNotification(
+          460,
+          10,
+          'Changed successfully!',
+          'Congrats! The salary was successfully changed.',
+          'success');
+        break;
+
+      case 0:
+      case 1:
+      case 3:
+        td.innerHTML = td.firstChild.value;
+
+        pushNotification(
+          460,
+          10,
+          'Changed successfully!',
+          'Congrats! The employee data was successfully changed.',
+          'success');
+        break;
+    }
+  } else {
+    if (td.firstChild.value === '') {
+      td.innerHTML = editingArea.data;
+
+      return;
+    }
+
+    return editStart(td);
+  }
+
+  editingArea = null;
+};
+
+function changeIsCorrect() {
```

Comment:
```suggestion
function resetIsCorrect() {
```
since you always set the same (initial) value